docs-src/devguide.md

```diff
@@ -14,6 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
+## Developing Service APIs
+
+You must have a working [Go environment] and then clone the repo:
+
+```
+mkdir -p $GOPATH/src/sigs.k8s.io
+cd $GOPATH/src/sigs.k8s.io
+git clone https://github.com/kubernetes-sigs/service-apis
+cd service-apis
```

Comment:
One https://github.com/kubernetes-sigs/service-apis/pull/352 merges we may need to update this to indicate that this also works outside of GOPATH.

/cc @jpeach